packages/driver/src/cy/commands/navigation.ts

```diff
@@ -588,7 +588,7 @@ export default (Commands, Cypress, cy, state, config) => {
     } catch (e) {} // eslint-disable-line no-empty
   })
 
-  Cypress.multiDomainCommunicator.on('visit:url', ({ url }) => {
+  Cypress.multiOriginCommunicator.on('visit:url', ({ url }) => {
```

Comment:
Would `crossOriginCommunicator` be more accurate or maybe we call this one `primaryOriginCommunicator` to go along with `specBridgeCommunicator`?

Reply:
I started with crossOriginCommunicator but walked it back because that makes it sound like it's only used for cross origin when we also use it for intra origin communcation too.

I like primaryOriginCommunicator but that kicks the can down the road for a name when we merge the two.

Reply:
you convinced me to go with primaryOriginCommunicator